packages/components/color-picker/src/color-picker.tsx

```diff
@@ -167,11 +168,12 @@ export const ColorPicker = forwardRef<ColorPickerProps, "input">(
               }}
             >
               {withSwatch ? <ColorPickerSwatch {...swatchProps} /> : null}
-
               <PopoverTrigger>
                 <ColorPickerField
                   h={h}
                   minH={minH}
+                  aria-expanded="false"
+                  aria-controls={id}
```

Comment:
Please fix as below.
```suggestion
                  role="combobox"
                  aria-controls={id}
```